metis-authentication/metis-authentication-service/src/main/java/eu/europeana/metis/authentication/service/AuthenticationService.java

```diff
@@ -192,7 +202,7 @@ private boolean isPasswordValid(MetisUserModel metisUser, String passwordToTry)
    */
   public Credentials validateAuthorizationHeaderWithCredentials(String authorization)
       throws GenericMetisException {
-    if (StringUtils.isEmpty(authorization)) {
+    if (StringUtils.isBlank(authorization)) {
```

Comment:
The `String` class in java 11 has a `.isBlank()` method. This way we don't have to use an external library. There are many occurrences in this class. Or perhaps you're allowing for the string to be null?

Reply:
I'm safe guarding from null too.
I could do the same with Objects.isNull() && String.isBlank().
What do you prefeer?

Reply:
This is a good question. I don't know what I prefer. Is it the only reason to include the library in this module? If it isn't, perhaps for the time being we should just keep using `StringUtils`.

Reply:
I understand the point but I'm leaving it as is, I think the commons.lang3 is very common library and we use it everywhere. Even if we use one function of it, we use it so often and we know exactly what it does.. Creating/duplicating the functionality ,takes us away from that security, besides the library is 500kb.

I think this can be of broader discussion.. We could do what you are suggesting, if we do this everywhere, which could make sense. Leaving it out of this pr